### DISCUSSION TITLE
We need an update to tgtg-python 0.18.0

### DISCUSSION THREAD

**Discussion by wimpie007** [Chouffy/home_assistant_tgtg#142] [General]
404 problem with the current version.

tgtg-python 0.18.0 is just released. Solves the problem

**Comment by tjorim** [comment]
Please test https://github.com/Chouffy/home_assistant_tgtg/releases/tag/5.6.0 and let me know if it works.

**Reply by wimpie007** [reply]
works flawless. testing it for some hours already... (pre-release😉)
BIG THANKS!
